src/usr/local/www/status_dhcp_leases.php

```diff
@@ -469,6 +502,7 @@ function remove_duplicate($array, $field) {
 			</thead>
 			<tbody>
 <?php
+if (count($dhcp_leases_subnet_counter)) {
```

Comment:
Please change this if statement to use ':' and 'endif;' instead of brackets to make it easy to read and follow style used in this block